docker-compose.yml

```diff
@@ -46,16 +46,16 @@ services:
     restart: always
     ports:
     - "40007:3000"
-  agar-io:
-    build: agar-io
-    restart: always
-    ports:
-    - "40008:3000"
+  #agar-io:
+  #  build: agar-io
+  #  restart: always
+  #  ports:
+  #  - "40008:3000"
   agar-io-cigar:
     build:
       context: agar-io/cigar
       args:
-        OGAR_SERVER: "localhost:40010"
+        OGAR_SERVER: "api.agar-io.games.fachschaften.org"
```

Comment:
In the default docker-compose.yml the argument should point to the local ogar server.
You may want to override some arguments in a production environment as described in the [readme](https://github.com/fsrmatheinfo/spieleabend#client-server-apps). Though, I figure I shall add more detailed dcumentation on deployment in production (just opened issue #5 for that).

Reply:
I threw these things out of the branch, as I said, it wasn't really intended as a PR :smile: